src/connectedk8s/azext_connectedk8s/_troubleshootutils.py

```diff
@@ -0,0 +1,1009 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from argparse import Namespace
+from pydoc import cli
+from kubernetes import client, config, watch, utils
+from binascii import a2b_hex
+from logging import exception
+import os
+import yaml
+import json
+import datetime
+from subprocess import Popen, PIPE, run, STDOUT, call, DEVNULL
+import shutil
+from knack.log import get_logger
+from azure.cli.core import telemetry
+import azext_connectedk8s._constants as consts
+logger = get_logger(__name__)
+# pylint:disable=unused-argument
+# pylint: disable=too-many-locals
+# pylint: disable=too-many-branches
+# pylint: disable=too-many-statements
+# pylint: disable=line-too-long
+
+
+diagnoser_output = []
+
+
+def create_folder_diagnosticlogs(time_stamp):
+
+    global diagnoser_output
+
+    try:
+
+        # Fetching path to current directory to create the arc diagnostic folder
```

Comment:
is this comment apt? it's not current directory right? can you correct it

Reply:
changed it to the user directory